examples/plot_mne_example.py

```diff
@@ -0,0 +1,223 @@
+
+
+#General imports
+import numpy as np
+import pandas as pd
+from matplotlib import cm, colors, colorbar
+import matplotlib.pyplot as plt
+
+# Import MNE, as well as the MNE sample dataset
+import mne
+from mne import io
+from mne.datasets import sample
+from mne.viz import plot_topomap
+
+# FOOOF imports
+from fooof import FOOOF, FOOOFGroup
+from fooof.analysis import *
+
+
+###################################################################################################
+# Load & Check MNE Data
+# ---------------------
+#
+# First, we will load the example dataset from MNE, and have a quick look at the data.
+#
+# The MNE sample dataset is a combined MEG/EEG recording with an audiovisual task, as
+# described `here <https://martinos.org/mne/stable/manual/sample_dataset.html>`_.
+#
+# For the current example, we are going to sub-select only the EEG data,
+# and analyze it as continuous (non-epoched) data.
+#
+
+###################################################################################################
+
+# Get the data path for the MNE example data
+raw_fname = sample.data_path() + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = sample.data_path() + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+
+# Load the file of example MNE data
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+###################################################################################################
+
+# Select EEG channels from the dataset
+raw = raw.pick_types(meg=False, eeg=True, eog=False, exclude='bads')
+
+###################################################################################################
+
+# Explicitly adding a default EEG average reference required for source localization
+raw.set_eeg_reference()
+
+###################################################################################################
+
+# Explore the data
+raw.info()
+
+###################################################################################################
+
+# Plot the channel locations and labels
+raw.plot_sensors(show_names=True)
+
+###################################################################################################
+
+# Creating epochs
+reject = dict(eeg=180e-6)
+event_id, tmin, tmax = {'left/auditory': 1}, -0.2, 0.5
+events = mne.read_events(event_fname)
+epochs = mne.Epochs(raw, events=events, event_id=event_id,
+				tmin=5, tmax=125, baseline=None, preload=True)
+
+###################################################################################################
+
+# Creating Power Spectra Densities
+psd, freqs = mne.time_frequency.psd_welch(epochs, fmin=1., fmax=50., n_fft=2000, n_overlap=250, n_per_seg=500)
+
+###################################################################################################
+# fooofgroup
+# ------------------
+#
+# The FOOOFGroup object is used to fit
+# FOOOF models across the power spectra. A list of FOOOFGroup objects is returned.
```

Comment:
I think 'A list of FOOOFGroup objects is returned.' is an outdated comment (as that is true of the `fit_fooof_3d` function, but doesn't seem to apply here?